keyboards/ergotravel/keymaps/default/keymap.c

```diff
@@ -0,0 +1,141 @@
+  ),
+
+  [_LOWER] = KC_KEYMAP(
+    //,----+----+----+----+----+----+----.     ,----+----+----+----+----+----+----.
+      GRAVE, 1  , 2  , 3  , 4  , 5  , A,           B,  6  , 7  , 8  , 9  , 0  ,DEL,
+    //|----+----+----+----+----+----+----|     |----+----+----+----+----+----+----|
+       TAB , F1 , F2 , F3 , F4 , F5 , F6 ,         D,  Y  ,MINS, EQL,LBRC,RBRC,BSLS,
+    //|----+----+----+----+----+----+----|     |----+----+----+----+----+----+----|
+       LSFT, F7 , F8 , F9 , F10, F11, F12,       SPC,  N  , M ,COMM,DOT ,SLSH,ENT ,
+    //|----+----+----+----+----+----+----.     .----+----+----+----+----+----+----|
+       LCTL,LGUI,LALT,ADJT,LOWR,SPC,                 SPC, RASE,LEFT, UP ,DOWN,RIGHT
+    //`----+----+----+--+-+----/----/               \----\----+----+----+----+----'
+  ),
+
+  [_RAISE] = KC_KEYMAP(
+    //,----+----+----+----+----+----+----.     ,----+----+----+----+----+----+----.
+       ESC , 1  , 2  , 3  , 4  , 5  , A,          B,  6  , 7  , 8  , 9  , 0  ,BSPC,
+    //|----+----+----+----+----+----+----|     |----+----+----+----+----+----+----|
+       TAB , Q  , W  , E  , R  , T  , C ,         D,  Y  , U  , I  , O  , P  ,DEL ,
+    //|----+----+----+----+----+----+----|     |----+----+----+----+----+----+----|
+       LSFT, Z  , X  , C  , V  , B  , A,        SPC,  N  , M  ,COMM,DOT ,SLSH,ENT ,
+    //|----+----+----+----+----+----+----.     .----+----+----+----+----+----+----|
+       LCTL,LGUI,LALT,ADJT,LOWR,SPC,                 SPC, RASE,LEFT, UP ,DOWN,RIGHT
+    //`----+----+----+--+-+----/----/               \----\----+----+----+----+----'
+  ),
+
+  [_ADJUST] = KC_KEYMAP(
+    //,----+----+----+----+----+----+----.     ,----+----+----+----+----+----+----.
+       ESC , F1 , F2 , F3 , F4 , F5 , F6,         F7,  F8, F9 ,F10 , F11, F12,BSPC,
+    //|----+----+----+----+----+----+----|     |----+----+----+----+----+----+----|
+       TAB , Q  , W  , E  , R  , T  , C ,         D,  Y  , U  , I  , O  , P  ,DEL ,
+    //|----+----+----+----+----+----+----|     |----+----+----+----+----+----+----|
+       LSFT, Z  , X  , C  , V  , B  , A,        SPC,  N  , M  ,COMM,DOT ,SLSH,ENT ,
+    //|----+----+----+----+----+----+----.     .----+----+----+----+----+----+----|
+       LCTL,LGUI,LALT,ADJT,LOWR,SPC,                 SPC, RASE,LEFT, UP ,DOWN,RIGHT
+    //`----+----+----+--+-+----/----/               \----\----+----+----+----+----'
+  )
+
+};
+
+   
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
+          PLAY_SONG(tone_qwerty);
+        #endif
+        persistent_default_layer_set(1UL<<_QWERTY);
```

Comment:
Audio code should be removed (see above). 

Also, it's better to use this function: `set_single_persistent_default_layer(_QWERTY);` as it handles the same thing, plus additional stuff like the audio.